google-postgresql.yml

```diff
@@ -30,24 +30,6 @@ provision:
     required: true
     type: string
     details: Google sql tier. This determines CPU and RAM resource.
```

Comment:
Is there an example of how the field should look like that we could add? Or a link to gcp docs?

Reply:
Yes good shout - I have added a link to the gcp docs.